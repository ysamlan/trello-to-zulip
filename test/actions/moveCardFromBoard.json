{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One",
      "shortLink": "b1slb1sl"
    },
    "boardTarget": {
      "id": "btidbtidbtidbtidbtidbtid",
      "name": "Board Target Name"
    },
    "card": {
      "id": "cidcidcidcidcidcidcidcid",
      "name": "Card Name"
    },
    "member": {
      "id": "mcidmcidmcidmcidmcidmcid",
      "name": "Member Creator Full Name"
    }
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "moveCardFromBoard"
}