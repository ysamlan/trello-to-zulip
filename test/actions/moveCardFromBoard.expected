Member Creator Full Name moved card [Card Name](https://trello.com/c/cidcidcidcidcidcidcidcid) from **Board One** to **Board Target Name**